tux/handlers/event.py

```diff
@@ -24,13 +24,34 @@ async def handle_harmful_message(self, message: discord.Message) -> None:
         if message.author.bot:
             return
 
-        stripped_content = strip_formatting(message.content)
+        stripped_content = strip_formatting(
+            message.content,
+        )  # pretty sure we have message.clean_content to replace strip_formatting, might be wrong though -Dainis
 
         if is_harmful(stripped_content):
             await message.reply(
                 "-# ⚠️ **This command is likely harmful. By running it, all directory contents will be deleted. There is no undo. Ensure you fully understand the consequences before proceeding. If you have received this message in error, please disregard it.**",
             )
 
+    def breaks_no_hello(self, message: discord.Message) -> bool:
+        if message.author.bot:
+            return False
+
+        support_channel_id = 1172312674298761216
```

Comment:
We should make this less ATL specific, could you make this be a configurable option?
aka what Sorcery tell you to do above.

Reply:
Will look into it